samples/drivers/led/lp5569/src/main.c

```diff
@@ -59,6 +60,26 @@ int main(void)
 
 			k_sleep(DELAY_TIME_BREATHING);
 		}
+
+		k_sleep(DELAY_TIME_ON);
+
+		/* Turn on all LEDs at once to demonstrate write_channels */
+		memset(ch_buf, 255, ARRAY_SIZE(ch_buf));
+		ret = led_write_channels(led_dev, 0, ARRAY_SIZE(ch_buf), ch_buf);
+		if (ret) {
+			return ret;
+		}
+
+		k_sleep(DELAY_TIME_ON);
+
+		/* Turn off all LEDs at once to demonstrate write_channels */
+		memset(ch_buf, 0, ARRAY_SIZE(ch_buf));
+		ret = led_write_channels(led_dev, 0, ARRAY_SIZE(ch_buf), ch_buf);
+		if (ret) {
+			return ret;
+		}
+
+		k_sleep(DELAY_TIME_ON);
```

Comment:
You need to update the README.rst file with the new behavior.

And finally you also need to document this new feature in the release-notes.

Reply:
Of course, sorry for missing this. I'll try to document it appropriately. Thanks!